source/feature/feature-template.html.md

```diff
@@ -37,7 +37,7 @@ Include you email address that you can be reached should people want to contact
 Expand on the summary, if appropriate. Explain the goal; the more details you can provide the better. 
```

Comment:
more details you can provide the better -> be as detailed as possible. (The instruction is clearer and more concise,)